src/installation/index.md

```diff
@@ -1,117 +1,127 @@
-- For bare-metal (`no_std`) applications: `riscv32imc-unknown-none-elf`
-- For applications that require `std`: `riscv32imc-esp-espidf`
+## Install Rust
+
+Make sure you have Rust installed. If not, see the instructions on the [rustup][rustup.rs-website] website.
+
+See also [alternative installation methods][rust-alt-installation].
 
-> #### A note in RISC-V `no_std` Rust targets.
+> **Note**: If you run Windows on your host machine, make sure you have installed one of the ABIs listed below. For more details, see the rustup book > [Chapter _Windows_][rustup-book-windows]
 >
-> There are [different flavors of RISC-V 32 target in Rust] covering the different [RISC-V extensions].
+> - **MSVC**: Recommended ABI, included in the list of `rustup` default requirements. Use it for interoperability with the software produced by Visual Studio.
+> - **GNU**: ABI used by the GCC toolchain. Install it yourself for interoperability with the software built with the MinGW/MSYS2 toolchain.
 
 
-The bare-metal targets can be installed by running:
+[rustup.rs-website]: https://rustup.rs/
+[rust-alt-installation]: https://rust-lang.github.io/rustup/installation/other.html
+[rustup-book-windows]: https://rust-lang.github.io/rustup/installation/windows.html
 
-```bash
-rustup target add riscv32imc-unknown-none-elf
-```
 
-For `std` applications, the `riscv32imc-esp-espidf` target is currently [Tier 3] and does not have pre-built objects distributed through `rustup`, therefore, it does not need to be installed as the `no_std` targets. Furthermore, `std` projects, also require:
- - The `-Z build-std` [unstable cargo feature], this [unstable cargo feature] can also be added to `.cargo/config.toml` of your project. Our [template projects], which we will later discuss, already take care of this.
- - [`LLVM`] installed.
- - [`ldproxy`] installed.
- - ESP-IDF (this will be installed by automatically by [`esp-idf-sys`]).
-
-At this point, you should be ready to build Rust applications for all the Espressif chips based on `RISC-V` architecture.
-
-[`espup`, a tool that will be covered later]: #espup
-[`rustup`]: https://rustup.rs/
-[Rust nightly toolchain]: https://rust-lang.github.io/rustup/concepts/channels.html#working-with-nightly-rust
-[component]: https://rust-lang.github.io/rustup/concepts/components.html
-[template projects]: ../writing-your-own-application/generate-project-from-template.md
-[unstable cargo feature]: https://doc.rust-lang.org/cargo/reference/unstable.html
-[`LLVM`]: https://llvm.org/
-[different flavors of RISC-V 32 target in Rust]: https://doc.rust-lang.org/nightly/rustc/platform-support.html#tier-2
-[RISC-V extensions]: https://en.wikichip.org/wiki/risc-v/standard_extensions
-[Tier 3]: https://doc.rust-lang.org/nightly/rustc/platform-support.html#tier-3
-[`esp-idf-sys`]: https://github.com/esp-rs/esp-idf-sys
+## RISC-V targets
 
-## Xtensa targets
+To build Rust applications for the Espressif chips based on `RISC-V` architecture, do the following
```

Comment:
```suggestion
To build Rust applications for the Espressif chips based on `RISC-V` architecture, do the following:
```

Reply:
Done for all comments about the semicolon.